jenkins-master/jobs/mozilla-central_update/config.xml

```diff
@@ -103,20 +80,22 @@ NSPR_LOG_FILE=http.log</propertiesContent>
       <project>scripts</project>
       <filter></filter>
       <target></target>
-      <selector class="hudson.plugins.copyartifact.StatusBuildSelector"/>
+      <selector class="hudson.plugins.copyartifact.StatusBuildSelector">
+        <stable>true</stable>
+      </selector>
     </hudson.plugins.copyartifact.CopyArtifact>
     <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
       <commandLine>python -u submission.py --test-type=update --build-state=running --repository=mozilla-central --revision=$REVISION --locale=$LOCALE --update-number=$UPDATE_NUMBER treeherder_venv</commandLine>
       <executeFromWorkingDir>false</executeFromWorkingDir>
     </hudson.plugins.xshell.XShellBuilder>
     <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
-      <commandLine>python runtests.py --type=update --installer-url=$INSTALLER_URL --update-channel=$CHANNEL --update-target-build-id=$TARGET_BUILD_ID</commandLine>
+      <commandLine>python -u runtests.py --test-type=update --repository=mozilla-central --installer-url=$INSTALLER_URL --update-channel=$CHANNEL --update-target-build-id=$TARGET_BUILD_ID</commandLine>
       <executeFromWorkingDir>false</executeFromWorkingDir>
     </hudson.plugins.xshell.XShellBuilder>
   </builders>
   <publishers>
     <hudson.tasks.ArtifactArchiver>
-      <artifacts>build/upload/*.*</artifacts>
+      <artifacts>build/http.log,build/upload/**</artifacts>
```

Comment:
This is only important for update tests and investigation of download problems of the MAR files.

So it will not work with a simple change in the job config. I would have to specify the path to the log file in a platform independent way which is not possible here. Option is to move the setting of the environment variables into the runtests.py script and have an extra section in config.py for additional env variables.